src/encoded/tests/features/search.feature

```diff
@@ -11,16 +11,23 @@ Feature: Search
 
     Scenario: Search Antibodies
         When I press "Materials & Methods"
+        And I wait for the content to load
         And I click the link to "/search/?type=AntibodyLot&status=released"
         And I wait for the content to load
         Then I should see at least 7 elements with the css selector "ul.result-table > li"
         And I should see at least 5 elements with the css selector "div.box.facets > div.orientation > div.facet"
 
-        When I click the link to "?type=AntibodyLot&status=released&targets.organism.scientific_name=Homo+sapiens"
+        When I click all elements with the css selector "[data-test='facetcontainer'] .facet__expander--header[aria-pressed='true']"
```

Comment:
This closes all opened facets. I want this not to fail if no facet was opened (the pre-opened facets could change at anytime). So, I made a new customstep and excluded the assert len(element) == 1.